dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/K8sNameSpaceServiceImpl.java

```diff
@@ -0,0 +1,326 @@
+            logger.error("namespace update to k8s error", e);
+            putMsg(result, Status.K8S_CLIENT_OPS_ERROR, e.getMessage());
+            return result;
+        }
+        // update to db
+        k8sNamespaceMapper.updateById(k8sNamespaceObj);
+
+        putMsg(result, Status.SUCCESS);
+        return result;
+    }
+
+    /**
+     * verify namespace and k8s
+     *
+     * @param namespace namespace
+     * @param k8s       k8s
+     * @return true if the k8s and namespace not exists, otherwise return false
+     */
+    @Override
+    public Result<Object> verifyNamespaceK8s(String namespace, String k8s) {
+        Result<Object> result = new Result<>();
+        if (StringUtils.isEmpty(namespace)) {
+            putMsg(result, Status.REQUEST_PARAMS_NOT_VALID_ERROR, Constants.NAMESPACE);
+            return result;
+        }
+
+        if (StringUtils.isEmpty(k8s)) {
+            putMsg(result, Status.REQUEST_PARAMS_NOT_VALID_ERROR, Constants.K8S);
+            return result;
+        }
+
+        if (checkNamespaceExistInDb(namespace, k8s)) {
+            putMsg(result, Status.K8S_NAMESPACE_EXIST, namespace, k8s);
+            return result;
+        }
+
+        putMsg(result, Status.SUCCESS);
+        return result;
+    }
+
+    /**
+     * delete namespace by id
+     *
+     * @param loginUser login user
+     * @param id        namespace id
+     * @return
+     */
+    @Override
+    public Map<String, Object> deleteNamespaceById(User loginUser, int id) {
+        Map<String, Object> result = new HashMap<>();
+        if (isNotAdmin(loginUser, result)) {
+            return result;
+        }
+
+        K8sNamespace k8sNamespaceObj = k8sNamespaceMapper.selectById(id);
+        if (k8sNamespaceObj == null) {
+            putMsg(result, Status.K8S_NAMESPACE_NOT_EXIST, id);
+            return result;
+        }
+
+        k8sClientService.deleteNamespaceToK8s(k8sNamespaceObj.getNamespace(), k8sNamespaceObj.getK8s());
+        k8sNamespaceMapper.deleteById(id);
+        putMsg(result, Status.SUCCESS);
+        return result;
+    }
+
+    /**
+     * check namespace name exist
+     *
+     * @param namespace namespace
+     * @return true if the k8s and namespace not exists, otherwise return false
+     */
+    private boolean checkNamespaceExistInDb(String namespace, String k8s) {
+        return k8sNamespaceMapper.existNamespace(namespace, k8s) == Boolean.TRUE;
+    }
+
+    /**
+     * use cpu memory create yaml
+     *
+     * @param k8sNamespace
+     * @return yaml file
+     */
+    private String genDefaultResourceYaml(K8sNamespace k8sNamespace) {
+        //resource use same name with namespace
+        String name = k8sNamespace.getNamespace();
+        String namespace = k8sNamespace.getNamespace();
+        String cpuStr = null;
+        if (k8sNamespace.getLimitsCpu() != null) {
+            cpuStr = k8sNamespace.getLimitsCpu() + "";
+        }
+
+        String MemoryStr = null;
```

Comment:
Please keep the naming styles consistent, the first letter of a variable name should be lowercase